src/Plugin/views/display_extender/PageHeaderDisplayExtender.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+namespace Drupal\localgov_core\Plugin\views\display_extender;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\Core\StringTranslation\TranslatableMarkup;
+use Drupal\views\Plugin\views\display_extender\DisplayExtenderPluginBase;
+use Drupal\views\Plugin\views\style\StylePluginBase;
+use Drupal\views\ViewExecutable;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Localgov page header display extender plugin.
+ *
+ * @ingroup views_display_extender_plugins
+ *
+ * @ViewsDisplayExtender(
+ *   id = "localgov_page_header_display_extender",
+ *   title = @Translation("Page header display extender"),
+ *   help = @Translation("Page header settings for this view."),
+ *   no_ui = FALSE
+ * )
+ */
+class PageHeaderDisplayExtender extends DisplayExtenderPluginBase {
+
+  use StringTranslationTrait;
+
+  /**
+   * The first row tokens on the style plugin.
+   *
+   * @var array
+   */
+  protected static $firstRowTokens;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    /** @var \Drupal\metatag_views\Plugin\views\display_extender\MetatagDisplayExtender */
```

Comment:
This variable hint references the Metatag display extender and isn't needed here anyway